x-pack/plugins/siem/public/management/pages/policy/view/ingest_manager_integration/configure_datasource.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo } from 'react';
+import { useLocation } from 'react-router-dom';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiEmptyPrompt, EuiText } from '@elastic/eui';
+import { useKibana } from '../../../../../../../../../src/plugins/kibana_react/public';
+import { LinkToApp } from '../../../../../common/components/endpoint/link_to_app';
+import { CustomConfigureDatasourceContent } from '../../../../../../../ingest_manager/public';
```

Comment:
are you importing files directly from `ingest_manager`?

Reply:
nvm, seems like its OK for now https://github.com/elastic/kibana/issues/64843

Reply:
Hmmm... 🤔  this was weird to me too. I will read up on what @oatkiller found with above link to understand better.

Reply:
AFAIK plugins are allowed to import from other plugin's root `/public` and `/server` dirs

Reply:
Yeah, I finally read up on it and understand the approach as to why `/public` + `/common` needs to be the path for exporting (better/smaller runtime bundles ++ more runtime re-usability) 👍